core/roboconf-dm/src/main/java/net/roboconf/dm/internal/autonomic/RulesParser.java

```diff
@@ -67,8 +67,7 @@ private RulesParser() {
 	public static Map<String,AutonomicRule> parseRules( ManagedApplication ma ) throws IOException {
 
 		Map<String,AutonomicRule> result = new HashMap<String,AutonomicRule> ();
-		String fileName = ma.getApplication().getName() + Constants.FILE_EXT_RULES;
```

Comment:
Compare with the code of ResourceUtils.java, line 70.
You do not look for the same file... In one case, it is **component-name.rules.cfg**, and in the other, it is **rules.cfg**.